js/apps/admin-ui/src/events/UserEvents.tsx

```diff
@@ -0,0 +1,476 @@
+                  name="client"
+                  label={t("client")}
+                  data-testid="client-searchField"
+                  isDisabled={!!client}
+                />
+                <FormGroup
+                  label={t("dateFrom")}
+                  fieldId="kc-dateFrom"
+                  className="keycloak__events_search__form_label"
+                >
+                  <Controller
+                    name="dateFrom"
+                    control={control}
+                    render={({ field }) => (
+                      <DatePicker
+                        className="pf-v5-u-w-100"
+                        value={field.value}
+                        onChange={(_, value) => field.onChange(value)}
+                        inputProps={{ id: "kc-dateFrom" }}
+                      />
+                    )}
+                  />
+                </FormGroup>
+                <FormGroup
+                  label={t("dateTo")}
+                  fieldId="kc-dateTo"
+                  className="keycloak__events_search__form_label"
+                >
+                  <Controller
+                    name="dateTo"
+                    control={control}
+                    render={({ field }) => (
+                      <DatePicker
+                        className="pf-v5-u-w-100"
+                        value={field.value}
+                        onChange={(_, value) => field.onChange(value)}
+                        inputProps={{ id: "kc-dateTo" }}
+                      />
+                    )}
+                  />
+                </FormGroup>
+                <TextControl
+                  name="ipAddress"
+                  label={t("ipAddress")}
+                  data-testid="ipAddress-searchField"
+                />
+                <ActionGroup>
+                  <Button
+                    data-testid="search-events-btn"
+                    variant="primary"
+                    type="submit"
+                    isDisabled={!isDirty}
+                  >
+                    {t("searchUserEventsBtn")}
+                  </Button>
+                  <Button
+                    variant="secondary"
+                    onClick={resetSearch}
+                    isDisabled={!isDirty}
+                  >
+                    {t("resetBtn")}
+                  </Button>
+                </ActionGroup>
+              </Form>
+            </DropdownPanel>
+          </FlexItem>
+          <FlexItem>
+            {Object.entries(activeFilters).length > 0 && (
+              <div className="keycloak__searchChips pf-v5-u-ml-md">
+                {Object.entries(activeFilters).map((filter) => {
+                  const [key, value] = filter as [
+                    keyof UserEventSearchForm,
+                    string | EventType[],
+                  ];
+
+                  const disableClose =
+                    (key === "user" && !!user) ||
+                    (key === "client" && !!client);
```

Comment:
Disable close button for user and/or client `Chip` if a value has been passed to the component. This shall prevent the user from removing the filter on a user or client details screen.